example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
This file shouldn't be in source control, please add it to the `.gitignore` list and remove it from the repo.